orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/orange/OrangeRecord.java

```diff
@@ -69,6 +70,9 @@ public interface OrangeRecord
     @NotNull
     LilacRecord lilac();
 
+    @NotNull
+    ImmuneEscapeRecord immuneEscape();
+
```

Comment:
We might want to make this field Nullable so we can handle tumor-only runs cleanly. If we don't null this for tumor-only runs then we could end up calling too many inactivations due to biallelic germline missense mutations that are harmless.

Or we'd have to do something smarter in tumor-only runs.

Reply:
Would prefer to leave as-is right now, as I think we should be able to say something sensible about immune escape in tumor-only. The current implementation is shaky in the end but maybe the problem is upstream (poor filtering of germline variants) or with the GIE implementation in general. Will let Joseph know